modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/SeriesServiceDatabaseImpl.java

```diff
@@ -169,12 +177,15 @@ public void deleteSeries(String seriesId) throws SeriesServiceDatabaseException,
           throw new UnauthorizedException(currentUser + " is not authorized to update series " + seriesId);
         }
       }
-      em.remove(entity);
+
+      entity.setModifiedDate(new Date());
+      entity.setDeletionDate(new Date());
```

Comment:
Maybe make sure they are actually identical?
```suggestion
      var date = new Date();
      entity.setModifiedDate(date);
      entity.setDeletionDate(date);
```